package.json

```diff
@@ -38,6 +38,7 @@
 		"classnames": "^2.5.1",
 		"csstype": "^3.1.3",
 		"dayjs": "^1.11.13",
+		"emoji-picker-react": "git+https://github.com/ZaneH/emoji-picker-react.git#main",
```

Comment:
I'm ready to transfer ownership of this repo to @spacebarchat
Once that's done we can update the Git URL here.

I tried starting the transfer process and got an error:

![image](https://github.com/user-attachments/assets/c523ca63-c454-4f6f-acb9-cb0b90e9d7f8)

Reply:
Send it to me

Reply:
> Repository transfer to Puyodead1 requested